### ISSUE TITLE
When multiple reasons are selected to reject a declaration, in Rejected action of record audit the reason in appearing "Others"

### ISSUE THREAD

**Issue by SyedaAfrida** [opencrvs/opencrvs-core#4176]
**Bug description:**
When multiple reasons are selected to reject a declaration, in Rejected action of record audit the reason in appearing "Others"

**Steps to Reproduce:**
1. Login to RA/ Registrar
2. Go to In progress/ Ready for review
3. Click Download> Assign> Update/ Review To any record
4. Click Reject
5. Select Multiple checkbox (reason)
6. Add Comment
7. Click Reject
8. Go to Requires updates tab
9. Click Download> Assign to that record
10. Click on the record
11. Click Rejected from History and observe reason

**Actual Result:**
The Reason is showing Other

**Expected result:**
It should show all the selected reasons

**Screen recording:**
https://images.zenhubusercontent.com/91778759/d97aa4da-b8a3-459e-9a1a-6f54a141f1af/multiple_reason_bug.mp4

**Tested on:**
https://login.farajaland-qa.opencrvs.org/?lang=en

**Comment by euanmillar** [comment]
We need to check the back end.  It may be that only one option is possible. If so, only one checkbox should be allowed to be selected

**Comment by jpye-finch** [comment]
Fixed with ticket https://app.zenhub.com/workspaces/opencrvs-core-5afeded64a54b06b54a969b2/issues/opencrvs/opencrvs-core/4286

**Comment by euanmillar** [comment]
Will be resolved by #4286

**Comment by jpye-finch** [comment]
@Zangetsu101 @CornelMaze we should be fixing this issue by working on 4286. As 4286 gets ride of the checkboxes/reasons

Personally @anny320 as i said. we should delete this ticket to avoid confusion.

**Comment by Zangetsu101** [comment]
Blocked by #4286

**Comment by Zangetsu101** [comment]
Closing this in favor of #4286
